administrator/components/com_media/resources/scripts/components/infobar/infobar.vue

```diff
@@ -21,11 +21,12 @@
                 <dd>{{ item.modified_date_formatted }}</dd>
 
                 <dt>{{ translate('COM_MEDIA_MEDIA_DIMENSION') }}</dt>
-                <dd v-if="item.width || item.height">{{ item.width }} x {{ item.height}}</dd>
+                <dd v-if="item.width || item.height">{{ item.width }}px * {{ item.height}}px</dd>
```

Comment:
```suggestion
                <dd v-if="item.width || item.height">{{ item.width }}px * {{ item.height }}px</dd>
```
Just for having it consistent.